fedimg/messenger.py

```diff
@@ -27,7 +27,7 @@
 """
 
 
-def message(topic, image_url, dest, status, extra=None):
+def message(topic, image_url, dest, status, extra=dict()):
```

Comment:
Oop, this will need a minor change.  It is a python programming anti-pattern to specify a [mutable default argument](http://docs.python-guide.org/en/latest/writing/gotchas/#mutable-default-arguments).

Here's the problem, illustrated:

```python
>>> def f(some_arg=dict()):
...     some_arg[random.random()] = 'this is now a thing'
...     print some_arg
...     
... 
>>> import random
>>> f()
{0.8328921394555624: 'this is now a thing'}
>>> f()
{0.8328921394555624: 'this is now a thing', 0.6220400583710248: 'this is now a thing'}
>>> f()
{0.8328921394555624: 'this is now a thing', 0.6220400583710248: 'this is now a thing', 0.5398246261893527: 'this is now a thing'}
>>> f()
{0.8328921394555624: 'this is now a thing', 0.6220400583710248: 'this is now a thing', 0.26180879892749087: 'this is now a thing', 0.5398246261893527: 'this is now a thing'}
```

See how the default value changes after every time you call the function?

I recommend doing it this way instead:

```python
>>> def f(some_arg=None):
...     some_arg = some_arg or dict()  # Put the default here, if it is a dict or a list
...     some_arg[random.random()] = 'this is now a thing'
...     print some_arg
... 
>>> f()
{0.6672781462320997: 'this is now a thing'}
>>> f()
{0.7093459128233877: 'this is now a thing'}
>>> f()
{0.6336043096870426: 'this is now a thing'}
>>> f()
{0.688688381954527: 'this is now a thing'}
```